lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -617,6 +617,29 @@ namespace Js
         return nullptr;
     }
 
+    void JavascriptStackWalker::WalkAndClearInlineeFrameCallInfoOnException()
+    {
+        // Walk the stack and when we find the first JavascriptFrame, we clear the inlinee's callinfo for this frame
+        // It is sufficient we stop at the first Javascript frame, because we still don't inline functions that have try-catch/try-finally into other functions
```

Comment:
>because we still don't inline functions that have try-catch/try-finally into other functions [](start = 67, length = 92)

Is there an assert somewhere which validate this?